### PR TITLE
fix: added pnpm format:fix in pre-commit hook

### PR DESCRIPTION
**Pull request by aarontravass**
As suggested in this [comment](https://github.com/tinyhttp/tinyhttp/pull/379#issuecomment-1537198782), `pnpm format:fix` has been added to husky pre-commit hook to avoid running it manually.

### CODE REVIEW

**Comment by aarontravass** [comment]
Linked CONTRIBUTING.md in README.md and added fork as first step instead of clone

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,14 +6,15 @@ Contributions are always welcome, here's an instruction of how to contribute.
...
+- Next, clone the repo:
 
 ```sh
 # git
-git clone https://github.com/tinyhttp/tinyhttp.git
+git clone https://github.com/{your_github_username}/tinyhttp.git
 
 # (or) hub
-hub clone tinyhttp/tinyhttp
+hub clone {your_github_username}/tinyhttp
```
Comment (talentlessguy) [inline CONTRIBUTING.md:17]: `hub` is an outdated CLI, use `gh` instead

File: .huskyrc.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "hooks": {
-    "pre-commit": "pnpm format && pnpm lint && pnpm build && pnpm test",
+    "pre-commit": "pnpm format:fix && pnpm format && pnpm lint && pnpm build && pnpm test",
```
Comment (talentlessguy) [inline .huskyrc.json:3]: there's no need to double run prettier

**Review by talentlessguy — CHANGES_REQUESTED** [review]

**Comment by talentlessguy** [comment]
@aarontravass yes that's a good idea

**Comment by aarontravass** [comment]
done

**Comment by talentlessguy** [comment]
wait why do we need 16.12+? what tool require it?

**Comment by aarontravass** [comment]
> wait why do we need 16.12+? what tool require it?

isn't it better to move to a newer version of node? security + performance?

**Comment by talentlessguy** [comment]
> > wait why do we need 16.12+? what tool require it?
> 
> isn't it better to move to a newer version of node? security + performance?

the "engine" field doesn't move your tools to a newer version, it requires you to use a newer version. The project  runs with no issue on 12.4+ so I'm not sure why an update is needed

**Comment by aarontravass** [comment]
> > > wait why do we need 16.12+? what tool require it?
> > 
> > 
> > isn't it better to move to a newer version of node? security + performance?
> 
> the "engine" field doesn't move your tools to a newer version, it requires you to use a newer version. The project runs with no issue on 12.4+ so I'm not sure why an update is needed

reverted the versions in package.json

**Comment by aarontravass** [comment]
> > > wait why do we need 16.12+? what tool require it?
> > 
> > 
> > isn't it better to move to a newer version of node? security + performance?
> 
> the "engine" field doesn't move your tools to a newer version, it requires you to use a newer version. The project runs with no issue on 12.4+ so I'm not sure why an update is needed

reverted the versions in package.json

**Comment by talentlessguy** [comment]
@aarontravass there's a merge conflict

**Comment by aarontravass** [comment]
> @aarontravass there's a merge conflict

oops, i missed it. just merged it

**Pull request merged**